src/init.cpp

```diff
@@ -475,6 +475,7 @@ std::string HelpMessage(HelpMessageMode mode)
         strUsage += HelpMessageOpt("-mocktime=<n>", "Replace actual time with <n> seconds since epoch (default: 0)");
         strUsage += HelpMessageOpt("-maxsigcachesize=<n>", strprintf("Limit sum of signature cache and script execution cache sizes to <n> MiB (default: %u)", DEFAULT_MAX_SIG_CACHE_SIZE));
         strUsage += HelpMessageOpt("-maxtipage=<n>", strprintf("Maximum tip age in seconds to consider node in initial block download (default: %u)", DEFAULT_MAX_TIP_AGE));
+        strUsage += HelpMessageOpt("-scrypt2peeriod=<n>", strprintf("Set custom block number to start scrypt2 period (should be higher than start of maintence period)"));
```

Comment:
It is a bad idea to allow a user to change such net parameter

Reply:
this parameter is in debug category, so it is obvious that it should be used only for testing purposes.
Also, it will not break the network if a  user changes it.

Reply:
ok